data/json/monsters/zed_misc.json

```
@@ -1083,7 +1083,7 @@
     "id": "mon_zombie_smoker",
     "type": "MONSTER",
     "name": { "str": "smoker zombie" },
-    "description": "A blackened and twisted naked human body, strips of flesh hang from its body, and it emits a constant haze of thick black smoke.",
+    "description": "A blackened and twisted naked human body. Strips of flesh hang from its frame, and it emits a constant haze of thick black smoke.",
```

```suggestion
    "description": "A blackened and twisted naked human body.  Strips of flesh hang from its frame, and it emits a constant haze of thick black smoke.",
```
Two spaces after period inside of " "